translate/pb/pb_util.go

```diff
@@ -0,0 +1,163 @@
+package pb
```

Comment:
General nit: It's important for reading and understanding the code that the filename describe the code it contains. "pb_util" just says "something to do with protobufs". This is both unhelpful for a reader of the code, and encourages people to dump anything related to protobufs into this file.

Consider a name that describes code that does exactly one thing, e.g. "pb_decoder".

Reply:
Changed to pb_decoder.go